pybamm/install_odes.py

```diff
@@ -75,38 +75,36 @@ def install_sundials(download_dir, install_dir):
     make_cmd = ["make", "install"]
     subprocess.run(make_cmd, cwd=build_directory, check=True)
 
-
 def update_LD_LIBRARY_PATH(install_dir):
-    # Look for current python virtual env and add export statement
-    # for LD_LIBRARY_PATH in activate script.  If no virtual env found,
-    # then the current user's .bashrc file is modified instead.
+    # Look for the current python virtual env and add an export statement
+    # for LD_LIBRARY_PATH in the activate script. If no virtual env is found,
+    # the current user's .bashrc file is modified instead.
 
     export_statement = f"export LD_LIBRARY_PATH={install_dir}/lib:$LD_LIBRARY_PATH"
 
     venv_path = os.environ.get("VIRTUAL_ENV")
     if venv_path:
         script_path = os.path.join(venv_path, "bin/activate")
     else:
-        script_path = os.path.join(os.environ.get("HOME"), ".bashrc")
+        if 'BASH' in os.environ:
+            script_path = os.path.join(os.environ.get("HOME"), ".bashrc")
+        if 'ZSH' in os.environ:
+            script_path = os.path.join(os.environ.get("HOME"), ".zshrc")
```

Comment:
I think this doesn't work either – I don't have these environment variables set on my machine. We should probably look for both of these rc files in the HOME directory and make changes to whichever file is found (both files if both are found)

Reply:
Ahh I see, looking accordingly into this.

Reply:
I hope now its alright.

Reply:
Yes it is, thanks!